cmd/git-remote-ipld/ipld_test.go

```diff
@@ -1,15 +1,19 @@
+//go:build !windows || !386
+// +build !windows !386
```

Comment:
Our stack _should_ build on 32 bit. Why is this necessary?

If there's a good reason, it would be good if that could be documented in a comment here.

Reply:
This is the comment I based the change on - https://github.com/ipfs-shipyard/git-remote-ipld/pull/35#discussion_r716137959

@aschmahmann do you have any more insight into this by any chance?

Reply:
I added a comment here: https://github.com/ipfs-shipyard/git-remote-ipld/pull/39/commits/d4c93371ed623dfa5c05059c41f353f16c58480b